awscli/customizations/cloudformation/yamlhelper.py

```diff
@@ -88,10 +101,12 @@ def yaml_dump(dict_to_dump):
     """
 
     yaml = ruamel.yaml.YAML(typ="safe", pure=True)
+    yaml.version = (1, 1)
     yaml.default_flow_style = False
     yaml.Representer = FlattenAliasRepresenter
     _add_yaml_1_1_boolean_resolvers(yaml.Resolver)
```

Comment:
If we are moving back to 1.1 is this still required?

Reply:
Nope, I tried removing it and it passed the tests, including `test_yaml_dump_quotes_boolean_strings`. Also tried `package` on a YAML file with various booleans and everything looked compliant with the 1.1 spec.